RR.App.Tests/Controllers/HRIS/EmployeeDateControllerUnitTests.cs

```diff
@@ -58,22 +56,9 @@ public async Task SaveEmployeeDateValidInputReturnsOkResult()
                                .Returns(Task.CompletedTask);
 
         var result = await _controller.SaveEmployeeDate(_employeeDateInput);
-
         Assert.IsType<OkResult>(result);
     }
 
-    [Fact]
-    public async Task SaveEmployeeDateExceptionThrownReturnsNotFoundWithMessage()
-    {
-        _employeeDateServiceMock.Setup(x => x.Save(It.IsAny<EmployeeDateDto>()))
-                               .ThrowsAsync(new Exception("An error occurred while saving employee date information."));
-
-        var result = await _controller.SaveEmployeeDate(_employeeDateInput);
-
-        var notFoundResult = Assert.IsType<NotFoundObjectResult>(result);
-        Assert.Equal("An error occurred while saving employee date information.", notFoundResult.Value);
-    }
-
```

Comment:
Fix: refer to  [UpdatePropertyRoleAccessReturnsSuccess line 130](https://github.com/RetroRabbit/RGO-Server/blob/df46ce432a2a34cf1c79aaf8b5eefd65c5731d24/RR.App.Tests/Controllers/HRIS/PropertyAccessControllerUnitTests.cs#L130) ,  [UpdatePropertyRoleAccessReturnsFail line 143](https://github.com/RetroRabbit/RGO-Server/blob/df46ce432a2a34cf1c79aaf8b5eefd65c5731d24/RR.App.Tests/Controllers/HRIS/PropertyAccessControllerUnitTests.cs#L143)

and the controller [UpdatePropertyRoleAccess line 45](https://github.com/RetroRabbit/RGO-Server/blob/df46ce432a2a34cf1c79aaf8b5eefd65c5731d24/RR.App/Controllers/HRIS/PropertyAccessController.cs#L45)

Include tests for success and failure. Fix the others as well below please.